.ci_scripts/run_build.sh

```diff
@@ -11,8 +11,17 @@ case "$BUILD_TARGET" in
 	cd build && make && make test && make install
 	echo "Creating disk image"
 	hdiutil create -volname Julius -srcfolder julius.app -ov -format UDZO julius.dmg
+	cd ..
+	;;
+"appimage")
+	cd build && make && make test
+	make DESTDIR=AppDir install
+	cd ..
+	./.ci_scripts/package_appimage.sh
 	;;
 *)
 	cd build && make && make test
+	make install
```

Comment:
Why run `make install` here? That usually requires root / admin privileges.

Reply:
Added by mistake while refactoring, then refactoring back. Note that I think it's a good thing in order to make sure that this command actually works. It's off-topic for this pull request however.